src/networks.json

```diff
@@ -950,6 +965,19 @@
     "explorer": "https://snowtrace.io",
     "start": 536483,
     "imageIPFS": "QmeS75uS7XLR8o8uUzhLRVYPX9vMFf4DXgKxQeCzyy7vM2"
+  },
+   "47805": {
+    "key": "47805",
+    "name": "REI Mainnet",
+    "chainId": 47805,
+    "network": "mainnet",
+    "multicall": "0x9eE9904815B80C39C1a27294E69a8626EAa7952d",
```

Comment:
Hi @bijianing97 Sorry for the delay, It looks great and works great, IS it possible for you to verify and publish these contracts so users can see the source code, Thanks :)
https://scan.rei.network/address/0x9eE9904815B80C39C1a27294E69a8626EAa7952d/contracts
https://scan-test.rei.network/address/0x9eE9904815B80C39C1a27294E69a8626EAa7952d/contracts

Reply:
No problem sir @ChaituVR , now you can see the contract code :
https://scan-test.rei.network/address/0x9eE9904815B80C39C1a27294E69a8626EAa7952d/contracts
https://scan.rei.network/address/0x9eE9904815B80C39C1a27294E69a8626EAa7952d/contracts